.github/workflows/book.yml

```diff
@@ -20,9 +33,26 @@ jobs:
         working-directory: book
       - run: cp -r book/book ../book
       - run: git checkout -- . && git clean -fd
-      - run: git fetch && git checkout gh-pages
       - run: test ! -d $(echo ${{github.ref}} | cut -d/ -f3) || rm -r $(echo ${{github.ref}} | cut -d/ -f3)
       - run: cp -r ../book $(echo ${{github.ref}} | cut -d/ -f3)
-      - run: git config --local user.name "github-actions[bot]" && git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
-      - run: git add $(echo ${{github.ref}} | cut -d/ -f3) && git commit --allow-empty -m "Docs build for SOF3/await-generator@${{github.sha}}"
-      - run: git push
+
+      - name: Setup Pages
+        uses: actions/configure-pages@v3
+      - name: Build with Jekyll
+        uses: actions/jekyll-build-pages@v1
```

Comment:
The book output is already HTML, why do we need another jekyll workflow here?

Reply:
To build the README.